phase/extender.go

```diff
@@ -366,12 +366,12 @@ func (e *Extender) dockerfilesFor(kind string, logger log.Logger) ([]extend.Dock
 }
 
 func (e *Extender) dockerfileFor(kind, extID string) (*extend.Dockerfile, error) {
-	dockerfilePath := filepath.Join(e.GeneratedDir, kind, launch.EscapeID(extID), "Dockerfile")
+	dockerfilePath := filepath.Join(e.GeneratedDir, launch.EscapeID(extID), fmt.Sprintf("%s.Dockerfile", kind))
```

Comment:
I think we need to switch on the Platform API version here so that the lifecycle will continue to work for older platforms

Reply:
Which one would be the correct platform API version? `v0.13`?

Reply:
Either sounds OK to me